packages/eslint-config/eslintrc.js

```diff
@@ -0,0 +1,180 @@
+    'no-redeclare': 'off',
+
+    /**
+     * TypeScript specific rules
+     * See https://github.com/typescript-eslint/typescript-eslint/tree/master/packages/eslint-plugin#supported-rules
+     */
+    '@typescript-eslint/array-type': 'off',
+    '@typescript-eslint/indent': 'off',
+    // '@typescript-eslint/no-explicit-any': 'off',
+    '@typescript-eslint/no-non-null-assertion': 'off',
+    '@typescript-eslint/explicit-function-return-type': 'off',
+    '@typescript-eslint/explicit-member-accessibility': 'off',
+    '@typescript-eslint/no-var-requires': 'off',
+    '@typescript-eslint/no-use-before-define': 'off',
+    '@typescript-eslint/no-object-literal-type-assertion': 'off',
+    '@typescript-eslint/no-parameter-properties': 'off',
+    '@typescript-eslint/no-angle-bracket-type-assertion': 'off',
+    '@typescript-eslint/prefer-interface': 'off',
+    '@typescript-eslint/no-namespace': 'off',
+    // '@typescript-eslint/no-unused-vars': 'off',
+    '@typescript-eslint/ban-types': 'off',
+    '@typescript-eslint/no-triple-slash-reference': 'off',
+    '@typescript-eslint/no-empty-interface': 'off',
+
+    /**
+     * The following rules are enforced to support legacy tslint configuration
+     */
+    // https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/ROADMAP.md
+    // Rules mapped from `@loopback/tslint-config/tslint.common.json
+    '@typescript-eslint/adjacent-overload-signatures': 'error', // tslint:adjacent-overload-signatures
+    '@typescript-eslint/prefer-for-of': 'error', // tslint:prefer-for-of
+    '@typescript-eslint/unified-signatures': 'error', // tslint:unified-signatures
+    '@typescript-eslint/no-explicit-any': 'error', // tslint:no-explicit-any
+
+    'no-unused-labels': 'error', // tslint:label-position
+    'no-caller': 'error', // tslint:no-arg
+    'no-new-wrappers': 'error', // tslint:no-construct
+    // 'no-redeclare': 'error', // tslint:no-duplicate-variable
+
+    'no-invalid-this': 'error',
+    '@typescript-eslint/no-misused-new': 'error',
+    'no-shadow': 'error', // tslint:no-shadowed-variable
+    'no-throw-literal': 'error', // tslint:no-string-throw
+
+    '@typescript-eslint/no-unused-vars': [
+      'error',
+      {
+        vars: 'all',
+        args: 'none', // none - do not check arguments
+        ignoreRestSiblings: false,
+      },
+    ], // tslint:no-unused-variable
+    'no-unused-expressions': 'error', // tslint:no-unused-expression
+    'no-var': 'error', // tslint:no-var-keyword
+    eqeqeq: ['error', 'smart'], // tslint:triple-equals: [true, 'allow-null-check', 'allow-undefined-check'],
+
+    // Rules mapped from `@loopback/tslint-config/tslint.build.json
+    '@typescript-eslint/await-thenable': 'error', // tslint:await-promise: [true, 'PromiseLike', 'RequestPromise'],
+
+    // https://github.com/xjamundx/eslint-plugin-promise
+    // tslint:no-floating-promises: [true, 'PromiseLike', 'RequestPromise'],
```

Comment:
Hmm, `no-floating-promise` is actually very useful! It detects situations where people forget to `await` a promise. 

Here is the tracking issue: https://github.com/typescript-eslint/typescript-eslint/issues/464 and pull request in progress: https://github.com/typescript-eslint/typescript-eslint/pull/495

Please rework the comment to point to those two links instead of https://github.com/xjamundx/eslint-plugin-promise.

I am ok to move forward with this PR even if `no-floating-promises` is not supported yet.